contracts/cw1-subkeys/src/contract.rs

```diff
@@ -119,11 +117,9 @@ where
         }
     }
     // Relay messages
-    let res = Response {
-        messages: msgs,
-        attributes: vec![attr("action", "execute"), attr("owner", info.sender)],
-        ..Response::default()
-    };
+    let res = Response::new()
+        .add_messages(msgs)
+        .add_attributes(vec![attr("action", "execute"), attr("owner", info.sender)]);
```

Comment:
wouldn't that be simpler with 2 add_attribute calls, since they are different V.

Not to be a stickler, but these contracts are like an example for other devs to follow

Reply:
Done almost everywhere now. We barely ever use `attr`.